client/my-sites/plugins/plugin-activate-toggle/test/index.jsx

```diff
@@ -43,6 +43,7 @@ describe( 'PluginActivateToggle', function() {
 	} );
 
 	beforeEach( function() {
+		this.timeout( 10 * 1000 );
```

Comment:
Yeah 10 was a guess. I'd rather have this way too lenient since circle is unpredictably slow. Once we have one runner, we can bump up the global timeout only when running under circle and remove these timeout hacks.